_client/livethere-app/src/components/summary/rentalSummary/RentalSummary.js

```diff
@@ -10,32 +10,29 @@ import Grid from "@material-ui/core/Grid";
 import axios from '../../../axios-wrapper';
 import useIsMountedRef from "../../../util/useIsMountedRef";
 import * as actionTypes from "../../../store/actions";
-import {connect} from "react-redux";
-
+import { connect } from "react-redux";
 
 const RentalSummary = (props) => {
   const [maxDistance, setMaxDistance] = useState(15);
   const [propertyType, setPropertyType] = useState('');
   const [bathCount, setBathCount] = useState('');
   const [bedCount, setBedCount] = useState('');
   const [summary, setSummary] = useState();
-  // TODO: append prediction to summary?
-  const [prediction, setPrediction] = useState();
   const isMountedRef = useIsMountedRef();
 
   useEffect(() => {
     fetchRentalSummary();
   }, [props]);
 
   const fetchRentalSummary = () => {
-    console.log('fetch rental summary data');
-
-    let params = {universityId: props.universityId};
-
-    if (maxDistance) {
+    let params = {
+      universityId: props.universityId,
+      postalCode: props.postalCode.slice(0, 3) + " " + props.postalCode.slice(3)
```

Comment:
I think this is a pretty sketchy way of doing it. Can we sanitize data in the backend instead? This is just a recommendation.

Reply:
Alright